src/app/grid-state/grid-state.component.html

```diff
@@ -56,12 +56,14 @@
             <igx-grid [id]="mcGridId" #mcGrid [igxGridState]="options" height="480px" [data]="localData"
                 displayDensity="compact" [allowFiltering]="true" [moving]="true">
                 <igx-column [resizable]="true" field="EmployeeID" [filterable]="false"></igx-column>
-                <igx-column-group [pinned]="false" header="Employee">
-                    <igx-column [sortable]="true" [resizable]="true" field="Country"></igx-column>
-                    <igx-column-group header="Employee Names">
-                        <igx-column [pinned]="false" [sortable]="true" [resizable]="true"
-                            field="FirstName"></igx-column>
-                        <igx-column [sortable]="true" [resizable]="true" field="LastName"></igx-column>
+                <igx-column-group header="Employee">
```

Comment:
Be careful not to change the formatting of a file, with the exception if you are changing it for better on purpose. Otherwise the file just appearas as changes and makes the PR harder to review (one should verify that there are no changes, only formatting)